libloki/storage.js

```diff
@@ -149,8 +150,14 @@
     );
   }
 
-  function savePairingAuthorisation(authorisation) {
-    return window.Signal.Data.createOrUpdatePairingAuthorisation(authorisation);
+  async function savePairingAuthorisation(authorisation) {
+    // Ensure that we have a conversation for all the devices
+    const conversation = await ConversationController.getOrCreateAndWait(
+      authorisation.secondaryDevicePubKey,
+      'private'
+    );
+    await conversation.setSecondaryStatus(true);
```

Comment:
Just wondering if `savePairingAuthorisation` could be called from the primary device while saving other device's authorisations? in which case the `setSecondaryStatus(true)` would not be correct..

Reply:
I think it would still be correct because I am doing
```js
    const conversation = await ConversationController.getOrCreateAndWait(
      authorisation.secondaryDevicePubKey,
      'private'
    );
```
So the conversation will always be the secondary device?

Reply:
Good point, never mind then :)